cdc/scheduler/internal/table_executor.go

```diff
@@ -30,33 +30,33 @@ type TableExecutor interface {
 	// if `isPrepare` is true, the 1st phase of the 2 phase scheduling protocol.
 	// if `isPrepare` is false, the 2nd phase.
 	AddTable(
-		ctx context.Context, tableID model.TableID, startTs model.Ts, isPrepare bool,
+		ctx context.Context, span tablepb.Span, startTs model.Ts, isPrepare bool,
 	) (done bool, err error)
 
 	// IsAddTableFinished make sure the requested table is in the proper status
-	IsAddTableFinished(tableID model.TableID, isPrepare bool) (done bool)
+	IsAddTableFinished(span tablepb.Span, isPrepare bool) (done bool)
 
 	// RemoveTable remove the table, return true if the table is already removed
-	RemoveTable(tableID model.TableID) (done bool)
+	RemoveTable(span tablepb.Span) (done bool)
```

Comment:
rename the function name?  it's span related func now

Reply:
Good point, thanks!